plugins/CoreUpdater/ReleaseChannel/LatestStable.php

```diff
@@ -29,11 +29,6 @@ public function getDescription()
         return Piwik::translate('General_Recommended');
     }
 
-    public function getDownloadUrlWithoutScheme($version)
-    {
-        return '://builds.matomo.org/matomo.zip';
```

Comment:
We need an update to our `OneClickUpdate` UI test:

>      Error: Failed to setup fixture Piwik\Tests\Fixtures\LatestStableInstall (error code = 1)

The [fixture](https://github.com/matomo-org/matomo/blob/9a3ef94df61682e7d543f3f8b438c368f6eb71a6/tests/PHPUnit/Fixtures/LatestStableInstall.php#L98) currently passes `null` as the version, and creates a URL that won't be available.